contentcuration/contentcuration/frontend/channelEdit/components/AssessmentEditor/AssessmentEditor.spec.js

```diff
@@ -250,43 +262,44 @@ describe('AssessmentEditor', () => {
 
   describe('on "Delete" click', () => {
     beforeEach(() => {
+      jest.clearAllMocks();
       const items = getItems(wrapper);
       clickDelete(items.at(1));
     });
 
-    it('emits delete item event with a correct item', () => {
-      expect(wrapper.emitted().deleteItem).toBeTruthy();
-      expect(wrapper.emitted().deleteItem.length).toBe(1);
-      expect(wrapper.emitted().deleteItem[0][0]).toEqual(ITEM2);
+    it('emits delete item event with a correct key', () => {
+      expect(listeners.deleteItem).toHaveBeenCalledWith(ITEM2);
+      expect(listeners.updateItems).toBeCalledTimes(1);
     });
 
     it('emits update item events with updated order of items after the deleted item', () => {
-      expect(wrapper.emitted().updateItem).toBeTruthy();
-      expect(wrapper.emitted().updateItem.length).toBe(2);
-      expect(wrapper.emitted().updateItem[0][0]).toEqual({
-        ...ITEM3,
-        order: 1,
-      });
-
-      expect(wrapper.emitted().updateItem).toBeTruthy();
-      expect(wrapper.emitted().updateItem.length).toBe(2);
-      expect(wrapper.emitted().updateItem[1][0]).toEqual({
-        ...ITEM4,
-        order: 2,
-      });
+      expect(listeners.updateItems).toHaveBeenCalledWith([
+        {
+          ...assessmentItemKey(ITEM1),
+          order: 0,
+        },
+        {
+          ...assessmentItemKey(ITEM3),
+          order: 1,
+        },
+        {
+          ...assessmentItemKey(ITEM4),
+          order: 2,
+        },
+      ]);
+      expect(listeners.updateItems).toBeCalledTimes(1);
     });
   });
 
   describe('on "Add question above" click', () => {
     beforeEach(() => {
+      jest.clearAllMocks();
       const items = getItems(wrapper);
       clickAddQuestionAbove(items.at(1));
     });
 
     it('emits add item event with a new item with a correct order', () => {
-      expect(wrapper.emitted().addItem).toBeTruthy();
-      expect(wrapper.emitted().addItem.length).toBe(1);
```

Comment:
Could this removed line be replaced by

```
expect(listeners.addItem).toBeCalledTimes(1);
```

like you do in other tests so that we're sure that no unnecessary calls are happening?